metrics/compute/compute.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package compute
+
+import (
+	"log"
+	"strings"
+
+	base "github.com/w3c/wptdashboard"
+	"github.com/w3c/wptdashboard/metrics"
+)
+
+type TestRunsStatus map[metrics.TestId]map[base.TestRun]metrics.CompleteTestStatus
+
+// Type for decision problem: "What does it mean for a test result to 'pass'"?
+type Passes func(*metrics.CompleteTestStatus) bool
+
+//
+// Passes functions
+//
+
+func OkAndUnknonwOrPasses(status *metrics.CompleteTestStatus) bool {
```

Comment:
Typo: `OkAndUnknonwOrPasses` => `OkAndUnknownOrPasses`

If you want to match the naming: `OkAndUnknownOrPass`